### ISSUE TITLE
Running erc20 script fails

### ISSUE THREAD

**Issue by danielgruesso** [coti-io/coti-sdk-typescript-examples#4]
Running yarn erc20 script fails with the following error:

```
yarn run v1.22.22
warning ../package.json: No license field
$ ts-node src/main.ts erc20
************* Onboarding user  0xe04f0C7eF0d73a935E19Db3AcB8f3Ae006478491  *************
************* Onboarded! created user key and saved into .env file *************
Running erc20 example...
/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/utils/errors.ts:694
            error = new Error(message);
                    ^
Error: could not decode result data (value="0x", info={ "method": "balanceOf", "signature": "balanceOf()" }, code=BAD_DATA, version=6.12.1)
    at makeError (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/utils/errors.ts:694:21)
    at assert (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/utils/errors.ts:715:25)
    at Interface.decodeFunctionResult (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/abi/interface.ts:916:15)
    at staticCallResult (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/contract/contract.ts:346:35)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async staticCall (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/contract/contract.ts:303:24)
    at async Proxy.balanceOf (/Users/daniel/projects/coti-sdk-typescript-examples/node_modules/ethers/src.ts/contract/contract.ts:351:41)
    at async erc20Example (/Users/daniel/projects/coti-sdk-typescript-examples/src/examples/erc20.ts:36:30)
    at async main (/Users/daniel/projects/coti-sdk-typescript-examples/src/main.ts:16:5) {
  code: 'BAD_DATA',
  value: '0x',
  info: { method: 'balanceOf', signature: 'balanceOf()' },
  shortMessage: 'could not decode result data'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by danielgruesso** [comment]
cc @gmesika-coti

**Comment by gmesika-coti** [comment]
issue was with the address of the pre-deployed contracts, they were pointing to the wrong addresses (prior to the reset of the devnet), fixed in commit d707987158fc9b70dbec103e78e517314997435d
